static/js/Router.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import { BrowserRouter, Route } from 'react-router-dom';
+import { Route, Router as ReactRouter } from 'react-router-dom';
 
 import App from './containers/App';
 import withTracker from './util/withTracker';
 
 export default class Router extends React.Component {
   props: {
-    store:  Store,
+    store:   Store,
+    history: Object,
   };
 
   render () {
-    const { store, children } = this.props;
+    const { store, children, history } = this.props;
 
     return <div>
       <Provider store={store}>
-        <BrowserRouter>
+        <ReactRouter history={history}>
```

Comment:
It makes it possible to pass in the `history` object, which is different for the unit tests